templates/test/ci/prow-topology/kustomization.yaml

```diff
@@ -5,9 +5,11 @@ resources:
   - ../../../flavors/clusterclass/cluster.yaml
   - cni-resource-set.yaml
```

Comment:
what is `cni-resource-set.yaml` ? is it installing calico for windows only or both

Reply:
I didn't notice that, let me check!

Reply:
@CecileRobertMichon It looks like the file [here](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/82a01b3392193d6bd2a088e668abe6a4ed7e3459/templates/test/ci/prow-topology/cni-resource-set.yaml) adds a config map for windows, but it's for ClusterResourceSet. In that case, are we safe to drop it?

Reply:
@Jont828 it seems to be installing Calico for the windows nodes (and there's a win md defined in the cluster) so I think we need to keep it for now